### PR TITLE
DO NOT MERGE - Include and pass version data for all nodes

### PR DESCRIPTION
**Pull request by murillodigital**
This PR adds a new version property to each node, and displays version data for each node/agent on the DS web UI as well as display the DS' own version.

DO NOT MERGE without coordinating as this will break everything, since out of date agents will be unable to properly unpack node data.

### CODE REVIEW

File: assignment-client/src/AssignmentClient.cpp [hunk 1]
```diff
@@ -41,8 +41,11 @@ const long long ASSIGNMENT_REQUEST_INTERVAL_MSECS = 1 * 1000;
 
 int hifiSockAddrMeta = qRegisterMetaType<HifiSockAddr>("HifiSockAddr");
 
-AssignmentClient::AssignmentClient(Assignment::Type requestAssignmentType, QString assignmentPool,
-                                   QUuid walletUUID, QString assignmentServerHostname, quint16 assignmentServerPort,
+AssignmentClient::AssignmentClient(Assignment::Type requestAssignmentType,
+                                   QString assignmentPool,
+                                   QUuid walletUUID,
+                                   QString assignmentServerHostname,
+                                   quint16 assignmentServerPort,
```
Comment (murillodigital) [inline assignment-client/src/AssignmentClient.cpp]: Not sure what the coding standard says in regards to putting the arguments below each other, I just think it's so much more readable.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 07bc3cf7a8d004aba2c1fdd4fb5379c68c13b6c3 available at: https://gist.github.com/e1755caef4deafcc781d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0ceb721e57319c6a0ca878d9809717863b37dc23 available at: https://gist.github.com/e1755caef4deafcc781d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7d864e1165049db2d5cc8d152419fb13d967799d available at: https://gist.github.com/e1755caef4deafcc781d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5347.dmg

**Comment by murillodigital** [comment]
this got very nasty conflicts after network refactor merge --- opening up a new clean PR

**Pull request closed without merging**